hal/src/main/java/edu/wpi/first/hal/can/CANExceptionFactory.java

```diff
@@ -14,7 +14,7 @@
   static final int ERR_CANSessionMux_NotAllowed = -44088;
   static final int ERR_CANSessionMux_NotInitialized = -44089;
 
-  @SuppressWarnings("MissingJavadocMethod")
+  @SuppressWarnings({"MissingJavadocMethod", "PMD.AvoidUncheckedExceptionsInSignatures"})
   public static void checkStatus(int status, int messageID)
       throws CANInvalidBufferException, CANMessageNotAllowedException, CANNotInitializedException,
           UncleanStatusException {
```

Comment:
Should this actually be suppressed? I removed the unchecked exception throws clauses from the shuffleboard classes, at least.

Reply:
I decided to remove the throws clause and add javadocs for the exceptions, which fixes the other warning suppression too.